docs/workflow.rst

```diff
@@ -0,0 +1,109 @@
+Automatic workflow
+==================
+Though **alchemlyb** is a library offering great flexibility in deriving free
+energy estimate, it also provide a easy pipeline that is similar to
+`Alchemical Analysis <https://github.com/MobleyLab/alchemical-analysis>`_ and a
```

Comment:
Use a reST link with alternative name
```reST
`Alchemical Analysis `_ and a ...


.. _Alchemical Analysis: https://github.com/MobleyLab/alchemical-analysis
```
especially as it is used multiple times. Btw, you can use 
```reST
The `displayed text 1 <link-target>`_ and `other text <link-target>`_ will link to the same
thing.

.. _`link-target`: https://example.com
```